<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="5">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,0,0,0,0,0">
          <item row="2" column="5">
           <widget class="QLineEdit" name="lineEdit_timer1">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>波特率</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLineEdit" name="lineEdit_timer0">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>定时器1 0X</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>定时器0 0X</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="comboBox_bolter">
            <item>
             <property name="text">
              <string>5Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>40Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>80Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>125Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>200Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>250Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>400Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>500Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>666Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>800Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1000Kbps</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>设备类型</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBox_devicetype">
            <item>
             <property name="text">
              <string>USBCAN  -I/I+</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>USBCAN  -2I/2I+</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_2">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>设备号</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QComboBox" name="comboBox_devicenum">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>CAN通道号</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QComboBox" name="comboBox_canindex">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>0</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>验收码 0X</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_acccode">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>00000000</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>屏蔽码 0X</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLineEdit" name="lineEdit_accmask">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>FFFFFFFF</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>模式</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="comboBox_mode">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>正常模式</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>只听模式</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>滤波方式</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QComboBox" name="comboBox_fiter">
            <item>
             <property name="text">
              <string>双滤波</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>单滤波</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>新波特率</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QComboBox" name="comboBox_newbolter">
            <item>
             <property name="text">
              <string>5Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>40Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>80Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>125Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>200Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>250Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>400Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>500Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>666Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>800Kbps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1000Kbps</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="5" column="5">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>设置新波特率</string>
            </property>
           </widget>
          </item>
          <item row="4" column="5">
           <widget class="QPushButton" name="pushButton_connect">
            <property name="text">
             <string>连接</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButton_fresh">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>更新</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_reset">
              <property name="text">
               <string>复位</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTableWidget" name="tableWidget">
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>节点地址</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>固件类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>固件版本</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>软件版本</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>执行命令</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>结果或原因</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string/>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButton_choosefile">
            <property name="text">
             <string>选择文件</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_filename"/>
          </item>
          <item row="1" column="7">
           <widget class="QPushButton" name="pushButton_upgrade">
            <property name="text">
             <string>升级</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>新固件路径</string>
            </property>
           </widget>
          </item>
          <item row="1" column="6">
           <widget class="QCheckBox" name="checkBox_crc16">
            <property name="text">
             <string>CRC16校验</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
